app/controllers/admin/base_controller.rb

```diff
@@ -0,0 +1,25 @@
+class Admin::BaseController < ActionController::Base
+  layout 'admin'
+  before_filter :authenticate_manager!
+  include AdminHelper
+
+  def welcome
+  end
+
+  private
+
+  def redirect_or_edit(obj, saved, notice = nil)
+    return redirect_to url_for_obj(obj), notice: notice_for(obj, notice) if saved
+    render 'edit'
+  end
+
+  def notice_for(obj, notice = nil)
+    notice ||= 'сохранен'

```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.